src/components/inputs/YearsCounter/YearsCounter.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import './styles.css';
+
+function YearsCounter({ setCount, count }: { setCount: any; count: number }) {
+  const [years, setYears] = useState('year');
+  const [registration, setRegistration] = useState('');
+
+  useEffect(() => {
+    changeYear();
+  }, [count]);
+
+  function subCount() {
+    if (count > 1) {
+      setCount(count - 1);
+    } else {
+      return;
+    }
+  }
+  function addCount() {
+    setCount(count + 1);
+  }
+  function changeYear() {
+    const date = new Date();
+    if (count > 1) {
+      setYears(`years`);
+    } else {
+      setYears(`year`);
+    }
+    const newYear = date.getFullYear() + count;
+    setRegistration(
+      `${date.toLocaleString('default', {
+        month: 'long',
+      })} ${date.getDate()}, ${newYear}`,
+    );
+  }
+  return (
+    <div className="yearsCounterContainer">
+      <p className="text white bold">Registration Period</p>
+      <div className="yearsCounter">
+        <button className="counterButton" onClick={() => subCount()}>
```

Comment:
it's good to be explicit in function names `subtractFromYears` or something similar